sdk/nodejs/runtime/stack.ts

```diff
@@ -55,10 +55,13 @@ class Stack extends ComponentResource<Inputs> {
      */
     public readonly outputs: Output<Inputs>;
 
+    private _init: () => Promise<Inputs>;
+
     constructor(init: () => Promise<Inputs>) {
-        super(rootPulumiStackTypeName, `${getProject()}-${getStack()}`, { init });
+        super(rootPulumiStackTypeName, `${getProject()}-${getStack()}`);
```

Comment:
yikes!

Reply:
Yeah - we should fix this separately from this PR.  This only triggered an error when I removed the change to explicitly ignore the input bag for components.  Regardless - this wasn't a great way to be doing things.